packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -234,6 +235,16 @@ export class SignIn extends BaseResource implements SignInResource {
     });
   };
 
+  validatePassword: ReturnType<typeof createValidatePassword> = (password, cb) => {
+    if (SignIn.clerk.__unstable__environment?.userSettings.passwordSettings) {
+      return createValidatePassword({
+        // TODO: Remove as any
```

Comment:
Is this something you wanted to do as part of this PR?

Reply:
This is how this functionality becomes part of the public api, and makes the feature available to developers who use clerk.

Reply:
I meant the TODO :D